hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestBlockTokens.java

```diff
@@ -206,7 +204,7 @@ public void blockTokenFailsOnExpiredSecretKey() throws Exception {
     StorageContainerException ex = assertThrows(StorageContainerException.class,
         () -> readDataWithoutRetry(keyInfo));
     assertEquals(BLOCK_TOKEN_VERIFICATION_FAILED, ex.getResult());
-    assertThat(ex).hasMessageContaining("Token can't be verified due to expired secret key");
+    assertTrue(ex.getMessage().contains("Token can't be verified due to expired secret key"));
```

Comment:
Please don't revert to the code prior to 0008d9ab086.

```suggestion
    assertThat(ex).hasMessageContaining("Token can't be verified due to expired secret key");
```